### DISCUSSION TITLE
scripthookvdotnet.log won't be created, but asiloader.log and scripthookv.log are normal, no errors

### DISCUSSION THREAD

**Discussion by Fernandess327** [scripthookvdotnet/scripthookvdotnet#1376] [Q&A]
Firstly, i would like to add that i have all of the requirements listed in this directory, and all of the requirements listed in ScripthookV download site as well, such as net framework above 4.8, and all of the Visual C++ required for both tools. I also have the latest version of  ScripthookV which i downloaded directly from their site that came out Dec 15th, 2023.

The problem i had at first was that the Native Trainer (From ScripthookV) menu wouldn't open up in my game when i pressed F4 (i play on steam desktop), after searching lots of threads and topics in lots of forums, i found out that scripthookdotnet was supposed to create a log file in the gta5 directory, just like scripthookv and the loader did. But mine is not, and i think this might be why the menu isn't working in game.

Lastly, i do have all of the files from the latest version available here in github (except the xml and txt). 

I have tried uninstalling every single trace of files about GTA 5 from my computer, including the mods, and installing again individually, still the problem persists. I have also tried to set all of the .exe files in gta directory to run as admin. Lastly, i have also tried installing GTA 5 in another directory to see if it would work, from scratch... and it didn't.

Can someone help me? I'm available to answer any questions and to collaborate with anyone to help solve my issue.

**Comment by kagikn** [comment]
For those wondering why I didn't respond here, it's because I tried to troubleshoot in 5Mods Discord. It turns out he had issues with having ScriptHookV run, so it would be safe to say that this issue is not relevant to SHVDN.
His message about SHV not working in his game in 5Mods Discord: https://discord.com/channels/318621297057988608/438244279262117888/1190825403166761081
